---
layout: paper
title: Orchestrated transcription of key pathways in <i>Arabidopsis</i> by the circadian clock
image: "../assets/img/ann-rev-growth.png"
authors: Harmer, Hogenesch, Straume, Chang, Han, Zhu, Wang, Kreps, and Kay
year: 2000
ref: Stacey Harmer, John Hogenesch, Marty Straume, Hur-Song Chang, Bin Han, Tony Zhu, Xun Wang, Joel Kreps, and Steve Kay (2000) <i>Science</i>
journal: "Science"
doi: http://science.sciencemag.org/content/290/5499/2110
---

# Abstract

Like most organisms, plants have endogenous biological clocks that coordinate internal events with the external environment. We used high-density oligonucleotide microarrays to examine gene expression in Arabidopsis and found that 6% of the more than 8000 genes on the array exhibited circadian changes in steady-state messenger RNA levels. Clusters of circadian-regulated genes were found in pathways involved in plant responses to light and other key metabolic pathways. Computational analysis of cycling genes allowed the identification of a highly conserved promoter motif that we found to be required for circadian control of gene expression. Our study presents a comprehensive view of the temporal compartmentalization of physiological pathways by the circadian clock in a eukaryote.

Stacey L. Harmer1, John B. Hogenesch2, Marty Straume3, Hur-Song Chang4, Bin Han4, Tong Zhu4, Xun Wang4, Joel A. Kreps4, Steve A. Kay1,2,*
