<!doctype html>
<html class="no-js" lang="en">
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Harmer Lab</title>

    <link rel="stylesheet" type="text/css" href="http://localhost:4000/HarmerLabWebSite/assets/css/feeling_responsive.css">

  

	<script src="https://ajax.googleapis.com/ajax/libs/webfont/1.5.18/webfont.js"></script>
	<script>
		WebFont.load({
			google: {
				families: [ 'Lato:400,700,400italic:latin', 'Volkhov::latin' ]
			}
		});
	</script>

	<noscript>
		<link href='https://fonts.googleapis.com/css?family=Inconsolata:400,700|Open+Sans:400,400i,700,700i' rel='stylesheet' type='text/css'>
	</noscript>


	<!-- Search Engine Optimization -->
<meta name="description" content="This is the website for the Harmer lab, located in the Department of Plant Biology at the University of California, Davis.&lt;/a">
	




<link rel="canonical" href="http://localhost:4000/HarmerLabWebSite/research/">

	<!-- Facebook Open Graph -->
<meta property="og:title" content="Harmer Lab">
<meta property="og:description" content="This is the website for the Harmer lab, located in the Department of Plant Biology at the University of California, Davis.&lt;/a">
<meta property="og:url" content="http://localhost:4000/HarmerLabWebSite/research/">
<meta property="og:locale" content="en_EN">
<meta property="og:type" content="website">
<meta property="og:site_name" content="Harmer Lab">



	

	<link type="text/plain" rel="author" href="http://localhost:4000/HarmerLabWebSite/humans.txt">

	

	
  <link rel="icon" sizes="32x32" href="http://localhost:4000/HarmerLabWebSite/assets/img/favicon-32x32.png" />
  <link rel="icon" sizes="192x192" href="http://localhost:4000/HarmerLabWebSite/assets/img/touch-icon-192x192.png" />
  <link rel="apple-touch-icon-precomposed" sizes="180x180" href="http://localhost:4000/HarmerLabWebSite/assets/img/apple-touch-icon-180x180-precomposed.png" />
  <link rel="apple-touch-icon-precomposed" sizes="152x152" href="http://localhost:4000/HarmerLabWebSite/assets/img/apple-touch-icon-152x152-precomposed.png" />
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="http://localhost:4000/HarmerLabWebSite/assets/img/apple-touch-icon-144x144-precomposed.png" />
  <link rel="apple-touch-icon-precomposed" sizes="120x120" href="http://localhost:4000/HarmerLabWebSite/assets/img/apple-touch-icon-120x120-precomposed.png" />
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="http://localhost:4000/HarmerLabWebSite/assets/img/apple-touch-icon-114x114-precomposed.png" />
  <link rel="apple-touch-icon-precomposed" sizes="76x76" href="http://localhost:4000/HarmerLabWebSite/assets/img/apple-touch-icon-76x76-precomposed.png" />
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="http://localhost:4000/HarmerLabWebSite/assets/img/apple-touch-icon-72x72-precomposed.png" />
  <link rel="apple-touch-icon-precomposed" href="http://localhost:4000/HarmerLabWebSite/assets/img/apple-touch-icon-precomposed.png" />
  <meta name="msapplication-TileImage" content="http://localhost:4000/HarmerLabWebSite/assets/img/msapplication_tileimage.png" />
  <meta name="msapplication-TileColor" content="#fabb00" />


	

</head>
<body id="top-of-page" class="page-fullwidth">
	
	<div class="top-bar stacked-for-medium">
  <div class="top-bar-title">
    <span data-responsive-toggle="responsive-menu" data-hide-for="medium">
      <button class="menu-icon dark" type="button" data-toggle></button>
    </span>
    <strong class="show-for-small-only">Menu</strong>
  </div>
  <div id="responsive-menu">



    <div class="top-bar-left">
      <ul  class="menu vertical medium-horizontal" data-responsive-menu="drilldown medium-dropdown">
        

              

    
          


    
            
              <li><a href="http://localhost:4000/HarmerLabWebSite/">Home</a></li>


    
            
          
        

              

    
          


    
            
              <li class="active"><a href="http://localhost:4000/HarmerLabWebSite/research/">Research</a></li>


    
            
          
        

              

    
          


    
            
              <li><a href="http://localhost:4000/HarmerLabWebSite/team/">Lab members</a></li>


    
            
          
        

              

    
          


    
            
              <li><a href="http://localhost:4000/HarmerLabWebSite/papers/">Publications</a></li>


    
            
          
        

              

    
          


    
            

              <li class="">
                <a href="http://localhost:4000/HarmerLabWebSite/links/">Links</a>

                  <ul class="menu vertical">
                    

                      

                      <li><a href="http://localhost:4000/HarmerLabWebSite/links/safety/">Safety</a></li>
                    

                      

                      <li><a href="http://localhost:4000/HarmerLabWebSite/links/protocols/">Protocols and Links</a></li>
                    

                      

                      <li><a href="http://localhost:4000/HarmerLabWebSite/links/seminars/">Meetings and Seminars</a></li>
                    
                  </ul>

              </li>
            
          
        

              

    
          


    
            
              <li><a href="http://localhost:4000/HarmerLabWebSite/contact/">Contact</a></li>


    
            
          
        
    
      </ul>
    </div>





      </ul>
    </div>
  </div>
</div>

	

	

<div id="masthead-with-background-color" style="background: ;">
	<div class="row">
		<figure class="small-12 columns">
			<img src="http://localhost:4000/HarmerLabWebSite/images/image8_with_welcome_sun2.png" alt="Harmer Lab">
		</figure><!-- /.small-12.columns -->
	</div><!-- /.row -->
</div><!-- /#masthead -->










	


<div class="row t30">
	<div class="medium-12 columns">
		<article>
			<header>
				
				<h1></h1>
			</header>

			

			<h2> Research</h2>

<h3>Background </h3>
<p>Most organisms possess an internal timer or circadian clock that allows them to regulate their physiology to better adapt to our continually changing world. These circadian clocks generate roughly 24 hour rhythms in physiology and behavior that are maintained even in the absence of environmental cues. Although the molecular components of circadian clocks are not conserved across higher taxa, in all organisms studied these clocks are cell autonomous oscillators and in eukaryotes are composed of complex transcriptional networks.</p>

<p>The Harmer lab uses forward and reverse genetics, genomics, biochemistry, and physiological studies to better understand the nature of the plant clock and how it helps shape plant adaptations to ever-changing environments. We use techniques ranging from ‘omics approaches to field-based studies to better understand how the plant clock works and how it promotes fitness in the natural environment.</p>

<h3>Molecular nature of the circadian oscillator</h3>

<div class="small-6 columns"><img src="/HarmerLabWebSite/assets/img/research/polly-model-full.png" />
</div>
<p><em>Model of the plant circadian clock, from <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3976767/">Hsu and Harmer (2013) Trends Plant Sci</a></em></p>

<p>Our model system for studies into the molecular nature of the plant clock is <em>Arabidopsis thaliana</em>. Using this powerful genetic model, we and others have identified a large number of plant-specific transcription factors that regulate each other’s expression via multiple feedback loops to form a complex transcriptional network.</p>

<div class="spacer"></div>

<p>Why is the plant clock so complex? The animal circadian network has far fewer components and provides reliable timekeeping. We have used genetics and computer modeling approaches to investigate this question, formulating a simplified model of the clock and testing the requirement for different classes of components in different environmental conditions.</p>

<div class="spacer"></div>

<div class="small-6 columns"><img src="/HarmerLabWebSite/assets/img/research/akiva-simple-clock-smlr.png" />
</div>
<p><em>Simplified model of the plant circadian clock, from <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6142266/">Shalit-Kaneh et al (2018) PNAS</a></em></p>

<p>Our results (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6142266/">Shalit-Kaneh et al (2018) <em>PNAS</em></a>) suggest that this high degree of network complexity provides rhythmic robustness across the wide range of environmental conditions that plants are subject to on a daily and seasonal basis. We are currently investigating the role of individual clock factors under specific environmental conditions.</p>

<div class="spacer"></div>

<p>We are also investigating how XCT, a protein conserved across eukaryotes (<a href="http://www.plantcell.org/content/plantcell/20/5/1244.full.pdf">Martin-Tryon and Harmer (2008) <em>Plant Cell</em></a>), links the plant circadian oscillator with other fundamental cellular processes. Intriguingly, not only can Arabidopsis <em>XCT</em> rescue growth phenotypes in fission yeast mutant for the <em>S. pombe</em> ortholog (<a href="http://embor.embopress.org/content/15/8/894.long">Anver et al. (2014) <em>EMBO reports</em></a>), but the fission yeast gene can also rescue Arabidopsis <em>XCT</em> mutants.</p>

<div class="small-6 columns"><img src="/HarmerLabWebSite/assets/img/research/xct-rescue.jpg" />
</div>
<p><em>Fission yeast Xap5 rescues Arabidopsis mutant for xct (Kumimoto et al, unpublished)</em></p>
<div class="spacer"></div>
<p><br /></p>

<p>We are currently investigating the biochemical function of XCT and determining the cellular pathways in which it acts.
<br /></p>

<h3>Circadian and environmental control of growth and development</h3>
<h5>Sunflower heliotropism</h5>

<p>In previous work, we found that the circadian clock ‘gates’ the sensitivity of Arabidopsis seedlings to auxin (<a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.0050222">Covington et al. (2007) <em>PLoS Bio</em></a>); that is, the degree of response to auxin differs according to the time of day. We then wondered in what circumstances it would be advantageous for plants to have the complex temporal regulatory network of the circadian clock linked to the auxin signaling network, a key regulator of spatial control of plant growth.</p>

<p>This led us to investigate a possible role for the plant clock in sunflower heliotropism, or solar tracking. Although many plants are robust solar trackers (<a href="https://www.cambridge.org/core/books/plant-canopies/diurnal-leaf-movements-and-productivity-in-canopies/0BCB482BCA3986A3156B6AEC150A2C47#">Ehleringer and Forseth (1988)</a>), heliotropism is particularly obvious in domesticated sunflower due to its long single stem. Remarkably, the leaves and stems of sunflower plants not only bend from east to west during the day, following the apparent movement of the sun, but also bend from west to east during the night in anticipation of sunrise. We demonstrated that the circadian clock plays an important role in solar tracking, especially in nightime reorientation.</p>

<iframe width="560" height="315" src="https://www.youtube.com/embed/GCRNHdGXTi4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>

<p><em>Animation of solar tracking sunflowers, summary of results in <a href="http://science.sciencemag.org/content/353/6299/587">Atamian et al (2018) Science</a>.</em></p>

<p>We found that the back-and-forth bending of young plants is driven by asymmetric growth on the east and west sides of the stem, with the east side growing faster during the day and the west side growing faster at night.</p>

<div class="small-6 columns">

  <p>We are particularly interested in understanding the molecular growth pathways controlling the different growth rates across this symmetric organ. Current results suggest that distinct growth pathways regulate daytime and nighttime bending movements, a possibility that we are eagerly investigating.</p>
  <div class="spacer"></div>
  <p><br />
<em>Asymmetrical growth across sunflower stems drives solar tracking movements <a href="">Atamian et al (2016), Science</a>.</em></p>
</div>
<div class="small-6 columns">
<img src="/HarmerLabWebSite/assets/img/research/stem-growth.jpg" />
</div>

<h5>Environmental and circadian control of flower development</h5>

<p>During our analysis of solar tracking in sunflower, we became intrigued by a possible role for the circadian clock in regulation of flower development and pollination. Although sunflower heads appear to be single flowers, they are actually composed of hundreds or even thousands of individual florets grouped together into a flower-like structure called a capitulum. The individual disk florets found in the centers of these capitula in sunflower and many other members of the <em>Compositae</em> are protandrous; that is, they are male one day and then female the next day, a reproductive strategy that promotes outcrossing.</p>

<p>Late-stage floret development progresses in  a stepwise manner from the outside of each disk in towards the middle. In domesticated sunflower, with its large capitulum, three or four rings of florets mature simultaneously on a given day. We have found that the circadian clock along with light and temperature signaling pathways regulates the precise timing of floret development and that this affects pollinator visits and reproductive fitness.</p>

<div class="small-6 columns"><img src="/HarmerLabWebSite/assets/img/research/bee-on-flower.png" />
</div>
<p><em>Bee collecting pollen from male-stage florets (Creux et al, unpublished)</em></p>

<div class="spacer"></div>
<p>We are currently investigating the molecular pathways controlling the timing of late-stage floret development in primarily outcrossing sunflower and comparing them to those governing floret development in its self-pollinating relative, lettuce.</p>

		</article>
	</div><!-- /.medium-12.columns -->
</div><!-- /.row -->




	
	<div class="row column">
  <div id="jump-top-top" class="text-right">
    <a href="#top-of-page"><svg width="22" height="22" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg"><path d="M1683 1331l-166 165q-19 19-45 19t-45-19l-531-531-531 531q-19 19-45 19t-45-19l-166-165q-19-19-19-45.5t19-45.5l742-741q19-19 45-19t45 19l742 741q19 19 19 45.5t-19 45.5z"/></svg></a>
  </div>




  <nav class="row">
       <div id="subfooter-left" class="small-12 medium-6 columns sans">
        <small><p>Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> based on the theme <a href="http://phlow.github.io/feeling-responsive/">Feeling Responsive</a>.</p></small>
        </div>


</footer>

	

	

<script src="http://localhost:4000/HarmerLabWebSite/assets/js/javascript.js"></script>



	<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-60112281-1', 'auto');
  ga('set', 'anonymizeIp', true);
  ga('send', 'pageview');

</script>





</body>
</html>

